stable/aws-for-fluent-bit/README.md

```diff
@@ -96,15 +116,81 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `elasticsearch.replaceDots` | Enable or disable Replace_Dots  | On |
 | `elasticsearch.suppressTypeName` | OpenSearch 2.0 and above needs to have type option being removed by setting Suppress_Type_Name On  | |
 | `elasticsearch.extraOutputs` | Append extra outputs with value | `""` |
-| `additionalOutputs` | add outputs with value | `""` |
-| `priorityClassName` | Name of Priority Class to assign pods | |
-| `updateStrategy` | Optional update strategy | `type: RollingUpdate` |
-| `affinity` | Map of node/pod affinities | `{}` |
-| `env` | Optional List of pod environment variables for the pods | `[]` |
-| `tolerations` | Optional deployment tolerations | `[]` |
-| `nodeSelector` | Node labels for pod assignment | `{}` |
-| `annotations` | Optional pod annotations | `{}` |
-| `volumes` | Volumes for the pods, provide as a list of volume objects (see values.yaml) |  volumes for /var/log and /var/lib/docker/containers are present, along with a fluentbit config volume |
-| `volumeMounts` | Volume mounts for the pods, provided as a list of volumeMount objects (see values.yaml) | volumes for /var/log and /var/lib/docker/containers are mounted, along with a fluentbit config volume |
-| `dnsPolicy` | Optional dnsPolicy | `ClusterFirst` |
-| `hostNetwork` | If true, use hostNetwork | `false` |
+
```

Comment:
Click the "view file" link in the PR. the formatting is entirely broken. I think its because of this newline.

Reply:
removed extra line